app/domain/operations/eligible/build_eligibility.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require "dry/monads"
+require "dry/monads/do"
+
+module Operations
+  module Eligible
+    # Operation to support eligibility creation
+    class BuildEligibility
+      include Dry::Monads[:do, :result]
+      include ::Operations::Eligible::EligibilityImport[
+                configuration: "eligibility_defaults"
+              ]
+
+      # @param [Hash] opts Options to build eligibility
+      # @option opts [<GlobalId>] :subject required
```

Comment:
is meaning of 'subject' universally known? I suspect it means varying types possible - perhaps an example entity type could be helpful.